README.md

```diff
@@ -59,7 +59,37 @@ spec:
 
 This will create a new Sandbox named `my-sandbox` running the image you specify. You can then access the Sandbox using its stable hostname, `my-sandbox`.
 
-For more complex examples, including how to use the extensions, please see the [examples/](examples/) and [extensions/examples/](extensions/examples/) directories.
+For a more programatic approach you can use the [SDK](https://github.com/kubernetes-sigs/agent-sandbox/blob/main/clients/python/agentic-sandbox-client/README.md) like the following:
+```python
+from agentic_sandbox import Sandbox
+
+# The SDK abstracts all YAML into a simple context manager
+try:
+    with Sandbox(
+        template_name="python-runtime-template",
+        namespace="ai-agents"
+    ) as sandbox:
+
+        print("--- Sandbox is Ready! ---")
```

Comment:
Can we remove these prints, as this makes the example more verbose than necessary, and the focus should be on showing the syntax.